File: DSA/Stuti_Jain_590012149/Day_16_Question_1.c
#include<stdio.h>
int ls(int arr[], int size, int target){
    int i;
    for(i=0; i<size; i++){
        if(arr[i]==target){
            return i;
        }
    }
    return -1;
}

int main(){
    int size, target, i;
    printf("Enter the number of elements:");
    scanf("%d", &size);
    int arr[size];
    printf("Enter the elements:\n", size);
    for(i=0; i<size; i++){
        scanf("%d", &arr[i]);
    }
    printf("Target:");
    scanf("%d",&target);
    int index=ls(arr, size,target);
    if(index!=-1){
        printf(" %d\n", index);
    }else{
        printf("Element not present in the array\n");
    }
    return 0;
}

File: DSA/Prabhjeet_590013286/Day16/Day_16_Question_1.py
def fn(arr, target):
    for i in range(len(arr)):
        if arr[i] == target:
            return i  
    return -1  

arr_input = input("Enter the array elements separated by spaces: ")
arr = list(map(int, arr_input.strip().split()))

target = int(input("Enter the target element to search for: "))

result = fn(arr, target)
if result != -1:
    print(f"Target found at index: {result}")
else:
    print("Target not found in the array.")
